.travis.yml

```diff
@@ -85,6 +88,11 @@ before_install:
     if [ "$TRAVIS_OS_NAME" = "linux" ]; then
       # setup virtual display for after_success target
       if [ "$gui" = "true" ]; then export "DISPLAY=:99.0" && /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 ; fi ;
+      
+      # Qt 5.9
+      PATH=/opt/qt59/bin:${PATH}
+      qmake_conf="$qmake_conf PKG_CONFIG_PATH=/opt/qt59/lib/pkgconfig::$PKG_CONFIG_PATH"
+      cmake_conf="$cmake_conf PKG_CONFIG_PATH=/opt/qt59/lib/pkgconfig::$PKG_CONFIG_PATH"
```

Comment:
I think you can do it with 1 colon instead of 2.

Reply:
Correct. It is an artifact of copy-pasting.